devel/litehtml/files/container.patch

```diff
@@ -1,13 +1,32 @@
-Fix include path in installed container file to find litehtml.h where it
-will be installed.
```

Comment:
You removed the comment that explains what the patch is for. I think I'd like to keep it.

Reply:
comment not addressed

Reply:
Restored.